tests/cephadm/test_service_stop.py

```diff
@@ -0,0 +1,31 @@
+from cli.ceph.orch.orch import Orch
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify service stop operation for Mon and Mgr service
+    """
+    base_cmd = "cephadm shell -- ceph"
```

Comment:
No need to create `base_cmd` here. Below is the code how orch command
```
from cli.cephadm.cephadm import CephAdm
CephAdm(node).ceph.stop(service_name)
```

Reply:
Done